freedomofpress/securedrop
# Css: Delete confirmation with css-only modal

## Status

Work in progress

## Description of Changes

Fixes #295.

Changes proposed in this pull request:
According to the discussions in #295 and #2506, I have implemented a CSS-only modal for confirmation of deletion of collections or documents. In this PR, I have replaced the js confirmation by the CSS confirmation modal for 
* multiple collection deletions
* single collection deletion
* multiple documents of a single collection deletion

## Testing

How should the reviewer test this PR?
Send a message as a whistleblower, log as a journalist mark the message and press `Delete`. A confirmation modal will appear with options to `Cancel` or `Delete`. You can click on `Cancel`, close icon or anywhere outside the modal to close the modal. On clicking `Delete`, the selection collection(s)/document(s) will get deleted.

## Checklist

### If you made changes to the app code:
